crates/nu-utils/standard_library/std.nu

```diff
@@ -7,50 +7,77 @@ def _assert [
     }
 }
 
+def _assertion-error [start, end, label, message?: string] {
+    error make {
+        msg: $"Assertion failed: ($message)",
```

Comment:
maybe we could only print the `message` when it is given and `Assertion failed.` when it is not?

something like
```diff
-        msg: $"Assertion failed: ($message)",
+        msg: ($message | default "Assertion failed."),
```

Reply:
the examples below would have to be modified though

Reply:
These are all assertion errors. I think there can be _additional_ information if there is a message. For example:

```
assert equal $code 301 "The http site is permanently redirected to https one"
```

It it fails, it is an assertion error, and the message helps debugging what was the original idea.

Reply:
yep!

so i'd say that, when a message is given, there is no need to prepend "Assertion failed:" just print the message :yum: 
but when there is no message given, a simple "Assertion failed." might be enough

in the end, it's the `Assertion failed:` when the message is empty that i find strange :open_mouth:

Reply:
As you wish.